packages/devextreme/js/__internal/grids/grid_core/sticky_columns/m_sticky_columns.ts

```diff
@@ -540,6 +563,32 @@ const draggingHeader = (Base: ModuleType<DraggingHeaderViewController>) => class
   }
 };
 
+const editorFactory = (Base: ModuleType<EditorFactory>) => class EditorFactoryStickyColumnsExtender extends Base {
+  public updateFocusOverlay($element: dxElementWrapper, isHideBorder = false): void {
+    if (!isHideBorder) {
+      const scrollable = this._rowsView.getScrollable();
```

Comment:
I'd suggest to extract updateFocusOverlayFixedPosition method

Reply:
Ok, I'll refactor the code in the next pull request for sticky columns.